scripts/build/build-hotrod-image.sh

```diff
@@ -132,12 +132,13 @@ if [[ "${runtime}" == "k8s" ]]; then
   fi
 
   echo '::group:: run on Kubernetes'
-  kustomize build ./examples/hotrod/kubernetes | kubectl apply -n example-hotrod -f -
-  kubectl wait --for=condition=available --timeout=180s -n example-hotrod deployment/example-hotrod
+  cd ./examples/oci
```

Comment:
can you move `cd` into the script itself? e.g. 
```
bash ./examples/oci/deploy-all.sh clean
```
and in the script `cd $(dirname $0)`